ap/attendance/management/commands/change_to_single_roll_system.py

```diff
@@ -0,0 +1,39 @@
+from accounts.models import TrainingAssistant
+from attendance.models import Roll
+
+from django.core.management.base import BaseCommand
+
+TAS = list(TrainingAssistant.objects.all())
+AMS = list(User.objects.filter(groups__name="attendance_monitors").exclude(groups__name="training_assistant").exclude(groups__name="dev"))
+
+class Command(BaseCommand):
+  # to use: python ap/manage.py change_to_single_roll_system
+  def _change_to_single_roll_system(self):
+    rolls = list(Roll.objects.all())
+    i = 0
+    while rolls:
+      roll = rolls[i]
+      trainee = roll.trainee
+      event = roll.event
+      date = roll.date
+
+      duplicates = Roll.objects.filter(trainee=trainee, event=event.id, date=date)
+      if duplicates.count() == 1:
+        dup = duplicates.first()
+        if dup.submitted_by != trainee:
+          dup.submitted_by = trainee
+          dup.save()
+      else:
+        for dup in duplicates:
+          if dup.submitted_by != dup.trainee and dup.submitted_by in AMS:
```

Comment:
What about just (this will take of rolls inputted by team monitors and HCs):
```suggestion
          if dup.submitted_by != dup.trainee:
```

Reply:
According to Sven HCs and team monitors can't submit audit rolls so we don't need to worry about that